.github/workflows/actions.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       - name: Set up Python 3
         uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
         with:
-          python-version: 3
+          python-version: 3.12
```

Comment:
Perhaps just adding some sort of comment here to why we explicitly set 3.12 here?

Reply:
done